cpp/CMakeLists.txt

```diff
@@ -44,7 +44,8 @@ set(CMAKE_CUDA_STANDARD 14)
 set(CMAKE_CUDA_STANDARD_REQUIRED ON)
 
 SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
-SET(CMAKE_INSTALL_RPATH "\$ORIGIN")
+SET(CMAKE_INSTALL_RPATH "\$ORIGIN"
+                        "\$ORIGIN/../..")
```

Comment:
We shouldn't be setting this RPath to `../..` since this will be installed in `lib`.

Reply:
done